app/Mail/Subscription/User/CancelledUserSubscriptionMail.php

```diff
@@ -41,7 +41,7 @@ public function __construct(User $user, string $reason = null)
     public function build()
     {
         return $this
-            ->from(['address' => 'hello@kanka.io', 'name' => 'Kanka Team'])
+            ->from(['address' => 'no-reply@kanka.io', 'name' => 'Kanka Team'])
```

Comment:
This is the reverse, no-reply doesn't exist and gets flagged as spam. I remembered wrong, it's other emails that are still being sent as no-reply instead of hello. You can revert this change and ignore the other places, I'll create a new ticket.